### PR TITLE
Reference reactor-ts as module

### PR DESCRIPTION
**Pull request by petervdonovan**
Companion PR: https://github.com/lf-lang/lingua-franca/pull/1322

### CODE REVIEW

File: src/core/port.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ export abstract class Port<T extends Present> extends Trigger {
     public isPresent() {
 
         Log.debug(this, () => "In isPresent()...")
-        Log.debug(this, () => "value: " + this.value);
+        Log.debug(this, () => "value: " + this.value?.toString);
```
Comment (lhstrh) [inline src/core/port.ts:37]: Shouldn't this be `toString()` (parentheses missing).

File: package.json [hunk 2]
```diff
@@ -84,7 +83,10 @@
         "url": "http://people.eecs.berkeley.edu/~marten"
     },
     "contributors": [
+        "Yunsang Cho",
+        "Wonseo Choi",
```
Comment (lhstrh) [inline package.json:87]:
```suggestion
        "Wonseo Choi",
        "Peter Donovan",
```

**Review by lhstrh — APPROVED** [review]
Looks good to me!

File: package.json [hunk 3]
```diff
@@ -1,52 +1,51 @@
...
-        "typescript": "^4.3.5"
+        "typedoc": "0.23.11"
     },
-    "files": [
-        "lib"
-    ],
     "scripts": {
         "check-types": "tsc",
-        "build": "rm -rf dist && babel src --out-dir dist --extensions .ts",
+        "install": "npx rimraf lib && npx babel src --out-dir lib --extensions .ts",
```
[the remark this answers is not in the document]
Comment (petervdonovan) [reply package.json:44]: I'm confused. Previously, we were using `rm`, which was not portable; this change **introduces** the use of `rimraf`.

File: src/core/port.ts [hunk 4]
```diff
@@ -34,7 +34,7 @@ export abstract class Port<T extends Present> extends Trigger {
     public isPresent() {
 
         Log.debug(this, () => "In isPresent()...")
-        Log.debug(this, () => "value: " + this.value);
+        Log.debug(this, () => "value: " + this.value?.toString());
```
Comment (suyourice) [inline src/core/port.ts:37]: Why isn't this part like `value.toString()` ?

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (lhstrh) [reply package.json:44]: Sorry -- misread the diff!

File: src/core/port.ts (hunk 4 above)
Comment (petervdonovan) [reply src/core/port.ts:37]: [The question mark handles the fact that `this.value` is nullable](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

Comment (suyourice) [reply src/core/port.ts:37]: Ah thank you. :)

**Pull request merged**